packages/ui/src/modal/list/TokenStats.tsx

```diff
@@ -74,8 +74,15 @@ const TokenStats: FC<Props> = ({ token, collection }) => {
           {
             id: 3,
             label: (
-              <Flex css={{ alignItems: 'center', gap: '$2', mr: '$1' }}>
-                <span>Highest Trait Floor</span>
+              <Flex css={{ alignItems: 'center', gap: '$2', mr: '$1', minWidth: '0' }}>
```

Comment:
This is great, good stuff. Can you also add this to the other labels in this component? Referring to the "Collection Floor", "Last Sale" and the "Creator Royalties".